pallets/messages/src/lib.rs

```diff
@@ -123,6 +124,7 @@ pub mod pallet {
 	/// A temporary storage of messages, given a schema id, for a duration of block period.
 	/// At the start of the next block this storage is cleared and moved into Messages storage.
 	/// - Value: List of Messages
+	/// Obsolete storage (will be removed in next storage migration)
```

Comment:
Migration not included to reduce the scope of this PR

Reply:
@aramikm as we don't have any values stored in it (since they are cleared on the next block), do we need a storage migration at all? Perhaps one to be safe that no additional information is in it?

Reply:
We might now need storage migration, its a good time to clean it up